src/lib/withReveal.js

```diff
@@ -10,10 +10,7 @@
 import React from 'react';
 
 function withReveal(WrappedComponent, effect) {
-  let refProp = undefined;
```

Comment:
Here the withReveal HOC of react-reveal was checking if the WrappedComponent is a styled-component. That was working for styled-components version <4 but now in version 5.3.3, styled-components don't expose innerRef of styledComponentId of type string.